lib/workers/repository/dependency-dashboard.ts

```diff
@@ -263,6 +285,22 @@ export async function ensureDependencyDashboard(
     return;
   }
   let issueBody = '';
+
+  if (configMigrationRes?.result === 'pr-exists') {
```

Comment:
```suggestion
  if (configMigrationRes.result === 'pr-exists') {
```

...and other if-branches below.